packages/terra-form-radio/tests/jest/RadioField.test.jsx

```diff
@@ -8,12 +8,23 @@ import Radio from '../../src/Radio';
 
 window.matchMedia = () => ({ matches: true });
 
+let userAgentGetter;
+beforeEach(() => {
+  userAgentGetter = jest.spyOn(window.navigator, 'userAgent', 'get');
+});
+
 it('should render a default radio field', () => {
   const radioField = (<RadioField legend="Default RadioField" />);
   const wrapper = shallowWithIntl(radioField);
   expect(wrapper).toMatchSnapshot();
 });
 
+it('should render radio field with div element or legend element', () => {
```

Comment:
```suggestion
it('should render radio field with div element for safari browser', () => {
```

Reply:
we need to add similar tests for edge and chrome browsers also

Reply:
i have made changes as per suggestions.